.github/workflows/build-and-publish.yml

```
@@ -17,13 +19,25 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Extract metadata (tags, labels) for Docker
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ${{ vars.DOCKER_ORG }}/${{ github.event.repository.name }}
+          images: |
+            ${{ vars.DOCKER_ORG }}/${{ github.event.repository.name }}
+            ghcr.io/${{ github.repository }}
+          tags: |
+            type=ref,event=branch  # Tag with branch name
+            type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'main') }}
```

```suggestion
            type=semver,pattern={{version}}
            type=semver,pattern={{major}}.{{minor}}
            type=semver,pattern={{major}}
            type=ref,event=branch  # Tag with branch name
            type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'main') }}
```
We have only recently started adding semver tags. Previously, we would simply use `type=ref,event=tag`.

---

Noted, Thanks Steve. I'll make the changes for the other the code so that it will be tagged by version's, instead of the branch. :)